cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,27 @@
 /// <reference types="cypress" />
 
-describe('Sign In page', () => {
-  it('should provide an ability to log in', () => {
+describe('Site page', () => {
+  beforeEach(() => {
+    cy.visit('https://react-redux.realworld.io');
+  });
+
+  cy.get('h1').should('contain.text', 'Sign In');
```

Comment:
The assertion 'cy.get('h1').should('contain.text', 'Sign In');' should be placed inside a test case (it block).